owenhallgren/stretch-project
# Feature/table status

## What does this PR do?

- This PR adds another table to our dashboard (Active and complete)
- adds ability to complete, remove reviews from our active table
- adds ability to undo a complete review in our dashboard tables.(sends back to active table)

## How should it be tested in the UI?

- Add a review (from home), complete a review(from dashboard), cancel a review(from dashboard), undo a review(from dashboard). Confirm all of these methods work with no errors or bugs.
